src/routes/projects/[projectId]/cases/+page.svelte

```
@@ -865,11 +923,76 @@
+								Delete
+							</button>
 						</div>
 					</div>
 				{/if}
 			</Dialog.Content>
 		</Dialog.Positioner>
 	</Portal>
 </Dialog>
+
+{#if showDeleteConfirm && testCaseToDelete}
+	<!-- svelte-ignore a11y_no_noninteractive_element_interactions -->
```

ESLint

[error] svelte-ignore comment is used, but not warned

(svelte/no-unused-svelte-ignore)